projects/components/src/assets/resources/en.properties

```diff
@@ -22,6 +22,10 @@ vcd.cc.export.all=Export All Columns
 vcd.cc.exporter.downloading=Downloading...
 vcd.cc.exporter.writing=Writing File...
 
+#Quick Search
```

Comment:
Do we really need this header? All the keys already say quicksearch

If so, please add a space between `#` and `Quick`